percona/controller/pgcluster/schedule.go

```diff
@@ -0,0 +1,120 @@
+package pgcluster
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"github.com/percona/percona-postgresql-operator/internal/controller/postgrescluster"
+	"github.com/percona/percona-postgresql-operator/internal/logging"
+	"github.com/percona/percona-postgresql-operator/internal/naming"
+	v2 "github.com/percona/percona-postgresql-operator/pkg/apis/pgv2.percona.com/v2"
+	"github.com/percona/percona-postgresql-operator/pkg/apis/postgres-operator.crunchydata.com/v1beta1"
+	"github.com/pkg/errors"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
```

Comment:
please separate external dependencies and internal packages

Reply:
https://github.com/percona/percona-postgresql-operator/pull/634/commits/ac0b22e5c698dbb0d93c86b033ac001ed9858bdb